### ISSUE TITLE
Fix TravisCI build

### ISSUE THREAD

**Issue by mikebski** [mikebski/logstash-filter-datepart#5]
TravisCI build is failing - see: https://travis-ci.org/mikebski/logstash-filter-datepart

Seems to be some sort of classpath problem with the LS stuff.  Also failing locally on my laptop

**Comment by mikebski** [comment]
@algestam - any ideas on this?  I don't completely understand the classpath issue with JRuby, I'm afraid.

**Comment by algestam** [comment]
Hi!

I have very little experience working with TravisCI but would like to help.

Tried to build it locally using docker and could at least replicate the problem. While searching for more information about the problem log4j problem I noticed that there are other logstash plugins that seem to have the same problem, i.e. https://travis-ci.org/logstash-plugins/logstash-output-statsd/builds/183236884.

There are other logstash plugins that builds without error (i.e. https://github.com/logstash-plugins/logstash-filter-geoip) but I haven't had time to examine the differences between these projects and this one.

One thing i noticed was that `s.platform = 'java'` is declared in logstash-filter-geoip but I'm unsure whether that makes any difference.
